src/WebJobs.Script/Host/AggregateFunctionMetadataProvider.cs

```diff
@@ -27,7 +27,7 @@ public class AggregateFunctionMetadataProvider : IFunctionMetadataProvider
         private IFunctionMetadataProvider _hostFunctionMetadataProvider;
 
         public AggregateFunctionMetadataProvider(
-            ILogger<AggregateFunctionMetadataProvider> logger,
```

Comment:
why is this type changed?

Reply:
I referred to other classes like FunctionMetadataManager and RPCFunctionInvocationDispatcher, they do not have typed Logger. As I was adding more tests for AggregateFunctionMetadataProvider, I noticed that the typed Logger is not allowing me to pass **TestLogger** which is also of type _ILogger_. So, I thought of just following the pattern used in other classes and their Test files and made it just _ILogger._